helper/src/components/portalnav.js

```diff
@@ -282,9 +285,15 @@ export default function PortalNav({ config }) {
   }
 
   function mergeState(tab, field, value, previewLink) {
-
     let updatevals
     let newFields = new Map()
+    if(tabState !== tab || fieldState !== field){
```

Comment:
lets discuss

Reply:
Resolved by combining both variables into one state object. Thanks for the feedback @khowling! :)